framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/GenericScenarioImpl.java

```diff
@@ -112,10 +113,20 @@ public void deploy() {
         logger.info("Waiting for Workbench deployment to become ready.");
         for (WorkbenchDeployment workbenchDeployment : workbenchDeployments) {
             workbenchDeployment.waitForScale();
+
+            for (Instance workbenchInstance : workbenchDeployment.getInstances()) {
```

Comment:
(Strictly optional) This looks kind of repetitive. Personally I'd create a utility method
```
public void logNodeNameOfAllInstances(Deployment deployment, String whatIsIt) {
     deployment.getInstances().forEach(
         instance -> {
            String instanceNodeName = project.getOpenShiftUtil().getPod(instance.getName()).getSpec().getNodeName();
            logger.info("{} name: {} ", whatIsIt, instanceNodeName);
         }
     );
```

and then call it like

`logNodeNameOfAllInstances("Workbench", workbenchDeployment);`

Reply:
Adjusted to remove duplicities